src/clojure_lsp/source_paths.clj

```diff
@@ -80,7 +80,8 @@
                            (remove nil?))
           deps-files-by-local-root (->> local-roots
                                         (map (fn [local-root]
-                                               (let [sub-deps-file (io/file root-path local-root "deps.edn")]
+                                               (let [local-root (shared/relativize-filepath local-root root-path)
+                                                     sub-deps-file (io/file root-path local-root "deps.edn")]
```

Comment:
Could you create a test for that?

Reply:
seems like CI is failing because of this change

Reply:
Right, should be fixed now with a new testcase.